src/kafka-file/languageservice/services/common.ts

```diff
@@ -0,0 +1,21 @@
+import { SerializationSetting } from "../../../client/serialization";
+import { CalleeFunction, ConsumerBlock, Node, NodeKind, ProducerBlock } from "../parser/kafkaFileParser";
+
+export function isProducerBlock(node: Node): node is ProducerBlock {
+  return node.kind === NodeKind.producerBlock;
+}
+
+export function isConsumerBlock(node: Node): node is ConsumerBlock {
```

Comment:
Move  this function in your inlineCommandActivation because it is used only here.